### PR TITLE
Fix application error.

### PR DESCRIPTION
**Pull request by gomorizsolt**
Issue #128 (there's also an explanation).

Let's use early-returns to break `renderLanguages()`.

### CODE REVIEW

**Comment by gomorizsolt** [comment]
> Is it just me or is it weird that a function is defined in the middle of the component logic?

I didn't find it weird. Though if it's confusing, let's move it to the top.

**Comment by gomorizsolt** [comment]
> Does !languages necessarily mean an error?

I'd like to avoid looping over falsy values even if there's not much chance of it. 

> How about an empty repo for example?

Empty repositories return empty objects which means they're converted to empty arrays.

(1) If the repository is empty, once the fetch is fulfilled, `data` becomes `{}`.

https://github.com/c-hive/dotdev/blob/de33b7dce3b887563b1ac3bea022c2943356820e/src/components/Projects/utils/GithubUtils.js#L54

(2) Calling the `filter()` method on an empty object returns an empty array.

https://github.com/c-hive/dotdev/blob/de33b7dce3b887563b1ac3bea022c2943356820e/src/components/Projects/utils/GithubUtils.js#L63

(3) Looping over an empty array with `map()` is obviously fine.

https://github.com/c-hive/dotdev/blob/de33b7dce3b887563b1ac3bea022c2943356820e/src/components/Projects/ProjectDisplayer/ProjectDisplayer.js#L62-L76

I hope this breakdown answers your question.

File: src/components/Projects/ProjectDisplayer/ProjectDisplayer.js [hunk 1]
```diff
@@ -35,29 +35,17 @@ const ProjectDisplayer = ({ org, repo }) => {
...
-    );
-  }
-
-  if (repoInfoErr) {
-    const errorMessage = `An error has occurred while loading the ${repo} Github project. Please try again later.`;
-
-    return <ErrorContainer>{errorMessage}</ErrorContainer>;
-  }
+  function renderLanguages() {
+    if (languagesErr || !languages) {
```
Comment (thisismydesign) [inline src/components/Projects/ProjectDisplayer/ProjectDisplayer.js:39]: Re https://github.com/c-hive/dotdev/pull/130#issuecomment-623947932

Then I just wonder if it's possible to have a falsey `languages` without a `languagesErr`

Comment (gomorizsolt) [reply src/components/Projects/ProjectDisplayer/ProjectDisplayer.js]: Eh, good point, I don't think so. I'd expect an error if `languages` was a falsy value.

https://github.com/c-hive/dotdev/pull/130/commits/7adfaaff53b3fba453ba544e8fb5df6f0d7a01a5

**Pull request merged**